Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -1785,7 +1785,7 @@ protected function convertParameterDataToArray(array $parametersInformation)
      * @param MethodReflection $method The parameter's method
      * @return array Parameter information array
      */
-    protected function convertParameterReflectionToArray(ParameterReflection $parameter, MethodReflection $method = null)
+    protected function convertParameterReflectionToArray(ParameterReflection $parameter, MethodReflection $method)
```

Comment:
We never ever call this protected method without a `MethodReflection` so...

Reply:
Since `ReflectionService` is not final and this method not private, this could still break things – it's very unlikely though

Reply:
True. I mean we can postpone this change to master too, not picky here. But I also don't see this super internal method problematic either.

Reply:
Yes, I agree and don't think it's necessary to postpone this